pytest.ini

```diff
@@ -30,6 +30,7 @@ addopts =
     --ignore-glob "tests/integration/sources/*"
     --ignore-glob "tests/integration/envs/*"
     --ignore-glob "examples/tutorials/output/*"
+    --ignore-glob "build/*"
```

Comment:
ignoring build folder since it conflicts with our normal folder when discovering tests.